test/dhcp/conf_parser_test.rb

```diff
@@ -356,6 +356,40 @@ def test_multiline_host_parser
     ]]], Proxy::DHCP::CommonISC::ConfigurationParser.new.conf.parse!(MULTILINE_HOST)
   end
 
+  HOST_WITH_MAC64 =<<EOHOST_WITH_MAC64
```

Comment:
If you use this version, can't you indent the end tag?
```suggestion
  HOST_WITH_MAC64 = <<-EOHOST_WITH_MAC64
```

Reply:
I can indent it regardless, whitespace must not break the parser. I don't care, honestly I'd rather avoid big refactoring because I would probably need to reindent the whole test.

Reply:
Good point that it's following the current convention in the file. :+1: for consistency within a file.